### PR TITLE
Add pr template for samples repo

### PR DESCRIPTION
**Pull request by brooke-hamilton**
# Description

Adds a PR template, based on the PR template in the radius repo.

## Type of change

- This pull request is a minor refactor, code cleanup, test improvement, or other maintenance task and doesn't change the functionality of Radius samples (issue link optional).

Fixes: N/A

### CODE REVIEW

File: .github/pull_request_template.md
Comment (ytimocin) [inline .github/pull_request_template.md:1]: Is this template going to be automatically picked up by each PR created after this PR gets merged in?

Comment (ytimocin) [reply .github/pull_request_template.md:1]: Now that I have checked the Radius repo, I think that (⬆️) is the case 😀

**Review by ytimocin — APPROVED** [review]

**Comment by ytimocin** [comment]
Should the base branch be **edge**?

File: .github/pull_request_template.md
Comment (brooke-hamilton) [reply .github/pull_request_template.md:1]: Yes, all PRs will use this template after the merge to the default branch.

**Comment by brooke-hamilton** [comment]
> Should the base branch be **edge**?

The base (default) branch for this repo is v0.36. Changes are PR'd to this branch then later merged to the `edge` branch when the next release is created. (This is my current understanding, but I'm happy to be corrected if I have this wrong.)

**Pull request closed without merging**